src/core/logic.py

```diff
@@ -1820,3 +1821,46 @@ def get_active_proposal_form():
         active_form = models.ProposalForm.objects.filter(in_edit=False).first()
 
     return active_form
+
+
+def get_file_mimetype(file_path):
+    """Returns a guessed mimetype for a given file path.
+
+    Args:
+        file_path (str): the path to the storage location of the file.
+
+    Returns:
+        str
+    """
+    mimetype_guess = mimetypes.guess_type(file_path)
+
+    try:
+        mimetype = mimetype_guess[0]
+        if not mimetype:
+            mimetype = 'unknown'
+    except IndexError:
```

Comment:
Is it possible `mimetypes.guess_type(file_path)` to return a non-tuple?
 - i.e. is this `try/except` necessary?

Reply:
I copied this pattern from the code that was already there and assumed the `try/except` dealt with some edge case. I'll look into it and see if `mimetype_guess` can return `None`, for example.

Reply:
Thanks for pointing this out. According to the docstring of the `guess_type` function, a tuple should always be returned, even if it's `(None, None)`.